homeassistant/components/stream/const.py

```diff
@@ -23,16 +23,16 @@
 SEGMENT_DURATION_ADJUSTER = 0.1  # Used to avoid missing keyframe boundaries
 # Each segment is at least this many seconds
 MIN_SEGMENT_DURATION = TARGET_SEGMENT_DURATION - SEGMENT_DURATION_ADJUSTER
+
 # -1.5 should put us in the middle of the second to last segment even with variable
```

Comment:
Optionals: Could also clarify this is a relative offset based on the segment duration. (though this still may not make sense out of context)